static/panes/ast-view.ts

```diff
@@ -227,7 +227,7 @@ export class Ast extends MonacoPane<monaco.editor.IStandaloneCodeEditor, AstStat
                     code.source.to.line < code.source.from.line + 100
                 ) {
                     for (let i = code.source.from.line; i <= code.source.to.line; ++i) {
-                        if (colours[i - 1] !== undefined) {
+                        if (!(i - 1 in colours)) {
```

Comment:
This is the only change I'm iffy about as it's more than just type changes, but eslint was complaining

Reply:
I think it'd be just `i - 1 in colours` without the negation

Reply:
Oops, you're right